content/en/flux/installation/configuration/air-gapped.md

```diff
@@ -63,4 +63,4 @@ flux bootstrap git \
 as the deploy key on your Git server in advance.
 
 For more information on how to use the `flux bootstrap git` command,
-please see the generic Git server [documentation](flux/installation/generic-git-server/).
+please see the generic Git server [documentation](../../bootstrap/generic-git-server/).
```

Comment:
I've used absolute links in the new docs, please change it here as well e.g. `/flux/installation/bootstrap/generic-git-server.md`

Reply:
Updated